replace_words.py

```diff
@@ -4,62 +4,134 @@
+            input_text = input_text.replace(old, new)
+        print input_text
+        return input_text
+
+    def write_to_file(self, replaced_text):
+        self.__open_file_to_read_or_write(self.__output_file,
+                                          opr='w+', text=replaced_text)
+
+
+class ArgParser:
+
+    def __init__(self):
+
+        parser = argparse.ArgumentParser(description='Replace words')
+        parser.add_argument("--input_file", dest='input_file',
+                            help='file contain text you want to modify',
+                            default='file1.txt')
+        parser.add_argument("--output_file", dest='output_file',
+                            help='file where you want to write updated text',
+                            default='file2.txt')
+        parser.add_argument("--from", dest='fromm',
+                            help='list of words you want to replace',
+                            default='no,sad,yesterday')
+        parser.add_argument("--to", dest='to',
+                            help='list of words you want to replace with',
+                            default='yes,happy,today')
+
+        self.arguments = parser.parse_args()
+
+    @property
+    def input_file(self):
+        return self.arguments.input_file
+
+    @property
+    def output_file(self):
+        return self.arguments.output_file
+
+    @property
+    def fromm(self):
+        return self.arguments.fromm
+
+    @property
+    def to(self):
+        return self.arguments.to
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Replace words')
-    parser.add_argument("--input_file", dest='input_file',
-                        help='file contain text you want to modify',
-                        default='file1.txt')
-    parser.add_argument("--output_file", dest='output_file',
-                        help='file where you want to write updated text',
-                        default='file2.txt')
-    parser.add_argument("--from", dest='fromm',
-                        help='list of words you want to replace',
-                        default='no,sad,yesterday')
-    parser.add_argument("--to", dest='to',
-                        help='list of words you want to replace with',
-                        default='yes,happy,today')
-    arguments = parser.parse_args()
-
-    input_file = arguments.input_file
-    output_file = arguments.output_file
-    from_list_arr = arguments.fromm.split(',')
-    to_list_arr = arguments.to.split(',')
-    input_text = open_file_to_read_or_write(input_file, opr='r')
-    print input_text
-    for old, new in zip(from_list_arr, to_list_arr):
-        input_text = input_text.replace(old, new)
-    print input_text
-    open_file_to_read_or_write(output_file, opr='w+', text=input_text)
+    argP = ArgParser()
+
+    MyWordsModifier = WordsModifier(argP.input_file, argP.output_file,
+                                    argP.fromm.split(','), argP.to.split(','))
```

Comment:
you should do it in the property not in here.

like this:
+    @property
+    def fromm(self):
+        return self.arguments.fromm.split(',')